venues/ICLR.cc/2018/Workshop/python/reviewers_intersect.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import openreview
+import csv
+import config
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('csv', help="input file with emails, first_name, last_name")
+parser.add_argument('datestring', help="append to output file name")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print "connecting to ", client.baseurl
+datestring = args.datestring
+
+## intersect reviewers file (ICLR Conference) and reviewers group (only ICLR Workshop)
+workshop_reviewers = client.get_group('ICLR.cc/2018/Workshop/Reviewers')
+## get all members that are in email form, get their profiles to get ~ids
+emails = []
+workshop_reviewers = workshop_reviewers.members
```

Comment:
small tip: you can do this in one line like so:

workshop_reviewers = client.get_group('ICLR.cc/2018/Workshop/Reviewers').members